src/pages/signInV2.vue

```diff
@@ -139,9 +139,32 @@ async function fetchAvailableLogins(authProvider: AuthProvider) {
 }
 
 async function storeUserInfoAndRedirect(
-  userInfo: GetInfoOutput & { hasMfa?: boolean; pk?: string }
+  userInfo: GetInfoOutput & {
+    hasMfa?: boolean
+    pk?: string
+  }
 ) {
   const storage = getStorage()
+  if ((userInfo.loginType as string) === 'firebase') {
```

Comment:
I don't understand this, why are we initializing Core here? Why do we only do this for Firebase?

Reply:
Can we handle this more generally? This is already pretty hacky.